### PR TITLE
Updated readme with basic project information

### PR DESCRIPTION
**Pull request by adberglund**
First crack at the readme. I fully expect this to change a lot over the next few weeks, particularly as we implement things like OpenShift, lots of these instructions change.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,12 +1,114 @@
...
+To activate the virtual environment run ::
+
+    pipenv shell
+
+Configuration
+~~~~~~~~~~~~~
+
+This project is developed using the Django web framework. Many configuration settings can be read in from a `.env` file. An example file `.env.example` is provided in the repository. To use the defaults simply ::
+
+    mv .env.example .env
```
Comment (chambridge) [inline README.rst]: Maybe `cp` instead of mv so there is no change in git status?

File: README.rst [hunk 2]
```diff
@@ -1,12 +1,114 @@
...
+
+
+Modify as you see fit.
+
+Database
+~~~~~~~~
+
+PostgreSQL is used as the database backend for SaltCellar. A docker-compose file is provided for creating a local database container. If modifications were made to the .env file the docker-compose file will need to be modified to ensure matching database credentials. Several commands are available for interacting with the database. ::
```
Comment (chambridge) [inline README.rst]: Could mention/link to running postgres locally? Can mention that non-container users can just call `make run-migrations`.

**Comment by chambridge** [comment]
Since Readme is included in the RTD your About section meets what I had in mind conceptually for #37. Closes #37.

**Comment by adberglund** [comment]
Closes #33

**Pull request merged**